src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -154,7 +154,7 @@ const DictionaryDetailCard = (props) => {
             </ul>
           </fieldset>
           <fieldset>
-            <legend>Released Version</legend>
+            <legend>{releasedVersions.length > 1 ? 'Versions' : 'Version'}</legend>
```

Comment:
Do you have tests for both "Version" and "Versions"?

Reply:
Yeah, they have both been tested.

Reply:
Can you point me to each?

Reply:
@dkayiwa , I have pointed it out.